.github/workflows/python.yml

```diff
@@ -0,0 +1,44 @@
+name: Python linters
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - pontoon/**.py
+      - .github/workflows/python.yml
```

Comment:
I suggest we also run the workflow whenever `requirements/lint.txt` changes

Reply:
Added